pkg/loki/loki.go

```diff
@@ -289,11 +289,13 @@ func (c *Config) Validate() error {
 	if err := c.LimitsConfig.Validate(); err != nil {
 		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid limits_config config"))
 	}
-	if err := c.IngestLimits.Validate(); err != nil {
-		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid ingest_limits config"))
-	}
-	if err := c.IngestLimitsFrontend.Validate(); err != nil {
-		errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid ingest_limits_frontend config"))
+	if c.IngestLimits.Enabled {
+		if err := c.IngestLimits.Validate(); err != nil {
+			errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid ingest_limits config"))
+		}
+		if err := c.IngestLimitsFrontend.Validate(); err != nil {
+			errs = append(errs, errors.Wrap(err, "CONFIG ERROR: invalid ingest_limits_frontend config"))
+		}
```

Comment:
Changed this to conditional to fix the conflict with `make validate-example-configs` looking for a valid ingest limits section.